README.md

```diff
@@ -143,6 +143,16 @@ licence:
    url: "http://www.wtfpl.net/txt/copying/"
 ```
 
+##### Share
+
+If you want enable the share content on social network, add key in **root** settings.yaml. For example: 
+By defaut you can share on facebook, twitter, pinterest, google+.
+
+```yaml
+share: True
```

Comment:
I know that both works but it's more common to use `true` with a lower `T` in yaml.